src/components/application_manager/include/application_manager/message_helper.h

```diff
@@ -617,20 +617,32 @@ class MessageHelper {
                                   bool available,
                                   ApplicationManager& app_mngr);
 
-  /*
+  /**
+   * @deprecated Deprecated in favor of method CreateMessageWithFunctionID
    * @brief Sends notification to HMI to stop audioPathThru
    *
    * @param connection_key  Application connection key
    *
    * @return TRUE on SUCCES otherwise return FALSE
    */
-  static bool SendStopAudioPathThru(ApplicationManager& app_mngr);
+  DEPRECATED static bool SendStopAudioPathThru(ApplicationManager& app_mngr);
 
   /**
+   * @deprecated Deprecated in favor of method CreateMessageWithFunctionID
    * @brief Sends UnsubscribeWayPoints request
    * @return true if UnsubscribedWayPoints is send otherwise false
    */
-  static bool SendUnsubscribedWayPoints(ApplicationManager& app_mngr);
+  DEPRECATED static bool SendUnsubscribedWayPoints(
+      ApplicationManager& app_mngr);
```

Comment:
These can be removed instead, due to the fact that the next release is a major release.